.github/workflows/push-e2e-runner.yaml

```
@@ -51,7 +52,8 @@ jobs:
       - name: Build and Push Image
         uses: docker/build-push-action@v5
         with:
-          context: .ibm/images/
+          context: .
+          file: .ibm/images/Dockerfile
           push: true
           tags: ${{ env.REGISTRY }}/${{ env.REGISTRY_IMAGE }}:${{ env.IMAGE_TAG }}
```

can we create a unique tag each time? 

```suggestion
          tags: |
            ${{ env.REGISTRY }}/${{ env.REGISTRY_IMAGE }}:${{ env.IMAGE_TAG }}
            ${{ env.REGISTRY }}/${{ env.REGISTRY_IMAGE }}:${{ env.IMAGE_TAG }}-${{ env.SHORT_SHA }}
```